web/src/routes/Events.jsx

```diff
@@ -51,7 +51,7 @@ export default function Events({ path, ...props }) {
   const [searchParams, setSearchParams] = useState({
     before: null,
     after: null,
-    cameras: props.cameras ?? 'all',
+    cameras: props.event ? '' : props.cameras ?? 'all',
```

Comment:
why does this matter? these search params are not being used when props.event is set

Reply:
It technically doesn't matter, but it makes it easier to get from the single-event view back to a full listing of the camera(s).  Otherwise you load the single event, but all cameras are selected so you have to toggle them off and back on.

The same is true for the labels/zones, but emptying those out too meant even more clicks to get to a usable list.

Reply:
I don't think this is the right approach. I think the searchParams should not be touched at all, and when a single event is selected there should be a button that shows which will clear the event query arg and go back to the default events screen

Reply:
Thanks for the suggestions.  I had this thought initially, but got stuck on having the event list properly reload.  Moving the event ID into searchParams made it easier to filter...just need to construct the link more explicitly outside of Frigate :)